<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FileSaving</class>
 <widget class="QDialog" name="FileSaving">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>782</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>591</width>
     <height>151</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <property name="verticalSpacing">
     <number>7</number>
    </property>
    <item row="1" column="2">
     <widget class="QPushButton" name="ChooseButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="text">
       <string>Выбрать</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="lineEdit"/>
    </item>
    <item row="2" column="1" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QPushButton" name="AcceptButton">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 10pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Подтвердить</string>
      </property>
      <property name="autoDefault">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="3" column="1" colspan="2">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item alignment="Qt::AlignmentFlag::AlignHCenter">
       <widget class="QLabel" name="label_2">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>40</height>
         </size>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="1" colspan="2" alignment="Qt::AlignmentFlag::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>40</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true">font: 700 16pt &quot;Segoe UI&quot;;</string>
      </property>
      <property name="text">
       <string>Укажите путь</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
